operator/pkg/apis/istio/v1alpha1/validation/validation.go

```diff
@@ -86,11 +89,42 @@ func deprecatedSettingsMessage(values *valuesv1alpha1.Values) string {
 		{"pilot.ingress", "meshConfig.ingressService, meshConfig.ingressControllerMode, and meshConfig.ingressClass", nil},
 		{"global.mtls.enabled", "the PeerAuthentication resource", nil},
 		{"global.mtls.auto", "meshConfig.enableAutoMtls", nil},
+		{"telemetry.v1.enabled", mixerDeprecatedMessage, false},
+		{"global.disablePolicyChecks", mixerDeprecatedMessage, false},
+		{"pilot.policy.enabled", mixerDeprecatedMessage, false},
```

Comment:
Is this the full list?  Can we check all the settings that match the call outs in the Mixer Removal Plan?

Reply:
It is not the full list, I only added the basic settings that can control mixer on/off. Because anyway other settings would require these settings change first. Wdyt?